doc/sphinx/source/tutorials/polarized_fits.rst

```diff
@@ -0,0 +1,122 @@
+.. _polarized:
+
+How to run a Polarized fit
+==========================
+
+The user should first refer to the :ref:`detailed guide <n3fit-usage>` on how to run
+a standard unpolarized PDF fit. Most of the steps in that guide are still apply
+and in the following section we only highlight the differences.
+
+
+Preparing a runcard and running a fit
+-------------------------------------
+
+The writing of a polarized fit runcard follows exactly the guideline described
+in :ref:`Preparing a fit runcard <prepare-fits>`. The user only need to modify 
+and adjust various entries.
+
+In polarized fits, the positivity of the polarized PDFs are enforced using as
+a boundary condition an unpolarized PDF set. The information on how such a
+constraint is enforced is defined under the key ``positivity_bound`` as follows:
+
+.. code-block:: yaml
+
+  # Define the unpolarized PDF set to be used as BC for positivity
+  ...
+  positivity_bound:
+    unpolarized_bc: NNPDF40_nnlo_pch_as_01180
+    n_std: 1.00 # Standard Deviation to be added as Shift
+  ...
+
+where ``pdfbc`` specifies the name of the unpolarized PDF set to be used as a
+boundary condition while ``stderr`` specifies the shift in terms of the standard
+deviation to be applied to the PDF central values. If ``stderr=0.0`` then the
+PDF central values will be used to constrain their polarized counterparts.
+
+Given that polarized fits require different fitting bases and different theory
+constraints, the fields under the ``fitting`` key require adjustments.
+Specifically:
+
+.. code-block:: yaml
+
+  ...
+  fitting:
+    fitbasis: EVOL_POL
+    sum_rules: TSR
+    basis:
+    - {fl: sng, trainable: false, smallx: [1.094, 1.118], largex: [1.46, 3.003]}
+    - {fl: g, trainable: false, smallx: [0.8189, 1.844], largex: [2.591, 5.697]}
+    - {fl: t3, trainable: false, smallx: [-0.4401, 0.9163], largex: [1.773, 3.333]}
+    - {fl: t8, trainable: false, smallx: [0.5852, 0.8537], largex: [1.533, 3.436]}
+  ...
+
+where ``TSR`` specifies that only sum rules on the :math:`T_3` and :math:`T_8`
+distributions. If any of these values are specified different the program will
+raise an error.
+
+.. note::
+
+   While the treatment of integrability follows exactly the same concept as in the 
+   default unpolarized fits, the treatment of positivity in the polarized case 
+   requires specific treatment. Similar to the unpolarised fits, the cost function 
+   used to enforce the positivity is defined by the following observable:
+  
+   .. math::
+     \chi_{\mathrm{tot}}^2 \rightarrow \chi_{\mathrm{tot}}^2+ \Lambda_{\rm POS} \sum_{k=1}^8 \sum_{i=1}^{n_i} \operatorname{ReLU}\left(-\mathcal{C}_k\left(x_i, Q^2\right)\right)
```

Comment:
It is really $n_i$ as it sums over the number of $x$-grid points. That equation has exactly the same structure as Eq. (3.10) of the NNPDF4.0 paper. But it is true it can be called anything ($n_x$ for example) but $n_k$.

Btw, thanks for the review @giacomomagni, the other comments have been addressed in [1eb11df](https://github.com/NNPDF/nnpdf/pull/1979/commits/1eb11dfca21395acac8736603bbc60aca2c35f56).

Reply:
Okay, thank for the clarification. I'd call it $n_x$, it's more clear and correct as $i$ is the variable of the sum.

Reply:
Done now!